dev/preview/workflow/preview/deploy-gitpod.sh

```diff
@@ -479,8 +472,12 @@ done
 # configurePayment
 #
 
-# 1. Read versions from docker image
-docker run --rm "eu.gcr.io/gitpod-core-dev/build/versions:$VERSION" cat /versions.yaml > /tmp/versions.yaml
+# 1. Read versions from the file system. We rely on `leeway dev/preview:deploy-dependencies` to create this file for us
+# Or from the docker file if it doesn't exist
+# shellcheck disable=SC2050,SC2057
+if [ ! test -f "/tmp/versions.yaml" ]; then
```

Comment:
We have referenced `/tmp/versions.yaml` quite a few places above this check, how about moving this check further up together with the check for `command -v installer`